src/main/java/chess/model/piece/Pawn.java

```diff
@@ -0,0 +1,60 @@
+package chess.model.piece;
+
+import chess.model.position.ChessPosition;
+import chess.model.position.Distance;
+
+import java.util.List;
+
+public class Pawn extends Piece {
+    private static final int DISPLACEMENT = 1;
+    private static final int INITIAL_SPECIAL_DISPLACEMENT = 2;
+
+    public Pawn(Side side) {
+        super(side);
+    }
+
+    @Override
+    public String getText() {
+        if (side.isWhite()) {
+            return "p";
+        }
+        return "P";
+    }
+
+    @Override
+    public List<ChessPosition> findPath(ChessPosition source, ChessPosition target, Piece targetPiece) {
+        checkValidTargetPiece(targetPiece);
+        Distance distance = target.calculateDistance(source);
+        validateForwardPath(source, targetPiece, distance);
+        if (canCrossMove(source, distance) || canDiagonalMove(targetPiece, distance)) {
+            return distance.findPath(source);
+        }
+        return List.of();
+    }
+
+    private void validateForwardPath(ChessPosition source, Piece targetPiece, Distance distance) {
+        if (targetPiece != null && canCrossMove(source, distance)) {
+            throw new IllegalArgumentException("타겟 위치에 기물이 존재하여 전진할 수 없습니다.");
+        }
+    }
+
+    private boolean canDiagonalMove(Piece targetPiece, Distance distance) {
+        return isPossibleDiagonal(distance) && targetPiece != null && !isSameSide(targetPiece);
+    }
```

Comment:
```suggestion
    private boolean canDiagonalMove(Piece targetPiece, Distance distance) {
        return isPossibleDiagonal(distance) 
            && targetPiece != null 
            && !isSameSide(targetPiece);
    }
```
boolean 을 나타내는 조건이 많아진다면 줄바꿈을 통해 가독성을 높일 수 있어요 :)

Reply:
오오 확실히 더 잘 읽히는거 같아요! 반영하겠습니다.